### PR TITLE
update the community page

### PR DESCRIPTION
**Pull request by FeynmanZhou**

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
> LGTM. Do we archive the meeting notes over the years and save GitHub somewhere @FeynmanZhou?

@toddysm Not yet. I add an archive of notes in this PR https://github.com/oras-project/community/pull/22

cc @oras-project/oras-www-maintainers

File: docs/community.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
-Want to reach the ORAS community and developers?
-We're very interested in feedback and contributions for other artifacts.
+Welcome to the ORAS Project community.
+
+- [Join us](https://slack.cncf.io/) at [CNCF Slack](https://cloud-native.slack.com) under the **#oras** channel
+- Join the weekly community meeting at 4:00 PM PST at [Zoom meeting room](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631. See [meeting notes](https://hackmd.io/P-O6n222TcSMoJgHmTTduw?view) for more details
+- Follow us on Twitter: [@orasproject](https://twitter.com/orasproject)
+- Subscribe us on [Youtube](https://www.youtube.com/@cncforasproject)
```
Comment (vsoch) [inline docs/community.md:6]:
```suggestion
- Subscribe us on [YouTube](https://www.youtube.com/@cncforasproject) 
```

Comment (FeynmanZhou) [reply docs/community.md]: @vsoch Thanks. Resolved!

File: docs/community.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
-Want to reach the ORAS community and developers?
-We're very interested in feedback and contributions for other artifacts.
+Welcome to the ORAS Project community.
+
+- [Join us](https://slack.cncf.io/) at [CNCF Slack](https://cloud-native.slack.com) under the **#oras** channel
+- Join the weekly community meeting at 4:00 PM PST at [Zoom meeting room](https://zoom.us/j/8079328631?pwd=UzN0NUlmbmcvTTN5L0o5VUQ2YndyQT09) with ID 807 932 8631. See [meeting notes](https://hackmd.io/P-O6n222TcSMoJgHmTTduw?view) for more details
```
Comment (TerryHowe) [inline docs/community.md:4]: can you maybe not post the zoom meeting here, but in slack?

Comment (FeynmanZhou) [reply docs/community.md:4]: Ah, I am curious why not show the Zoom meeting on the website?

Comment (TerryHowe) [reply docs/community.md:4]: Minimize zoom bombing

**Review by SteveLasker — APPROVED** [review]
LGTM

Archiving the hackmd notes to github, under the community repo was captured [here](https://github.com/oras-project/community/pull/22) by @FeynmanZhou

**Review by TerryHowe — APPROVED** [review]
LGTM

File: docs/community.md (hunk 2 above)
Comment (Asmit2952) [inline docs/community.md:4]: bi-weekly* now

**Pull request merged**